css-fonts-4/Overview.bs

```diff
@@ -825,7 +825,7 @@ Font style: the 'font-style!!property' property</h3>
             Matches against a font that is labeled as an italic face,
             or an oblique face if one does not exist.
 
-        <dt><dfn>oblique <<angle>>?</dfn>
+        <dt><dfn>oblique <<angle [-90,90]>>?</dfn>
```

Comment:
I did the same for `<time>` related ranges. I thought Tabatkins was approving unitless ranges but I have no hard feeling against it. I will adapt my value definition parser to whatever is decided.

> So long as it's clear, I think the range limitation is fine without units

Reply:
Right, I said "so long as it's clear" - 0 and infinity are fine. I don't think it's immediately clear that "90" is meant to be a `deg` value; non-trivial values should get units.

(A `<percentage>` would be fine without, since it's only got one "unit" possible.)